django_project/frontend/src/pages/Dashboard/MapLibre/Layers/ContextLayers/index.jsx

```diff
@@ -21,13 +21,14 @@ import React, { Fragment, useEffect } from 'react';
 import { centroid as turfCentroid } from '@turf/turf';
 import { useSelector } from "react-redux";
 import { hasLayer, hasSource, removeLayer, removeSource } from "../../utils";
-import { arcGisLayer, geojsonLayer, rasterTileLayer } from "../../LayerType"
+import {arcGisLayer, geojsonLayer, rasterTileLayer} from "../../LayerType"
```

Comment:
Could you change the rule format ? to prevent this changes

Reply:
done